_episodes/l1-04-managing_contributions.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Managing contributions to code"
+teaching: 0 # TBD
+exercises: 0 # TBD
```

Comment:
I think it is time to think on the timings, here. How long do you think this lesson will take?